src/Infrastructure/Changelog.php

```diff
@@ -65,6 +65,13 @@ public function close(Version $version): void
         );
     }
 
+    public function getReleaseNotes(Version $version): array
```

Comment:
Add PHPDoc tag to specify which elements array contains (for example, `string[]`, etc.).